windows/ReactTestApp.sln

```diff
@@ -29,6 +29,7 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Mso", "$(ReactNativeModuleP
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Shared", "$(ReactNativeModulePath)\Shared\Shared.vcxitems", "{2049DBE9-8D13-42C9-AE4B-413AE38FFFD0}"
 EndProject
+$(TestProject)
```

Comment:
What happens with this variable if the test project doesn't exist?

Reply:
It's replaced by empty string. 
It could be an option to remove this completely and build test project separately from the solution, but it would remove possibility to easily run tests with Test Explorer in VS.

Reply:
This is perfectly fine. I didn't see that it would be replaced with an empty string.